popsycle/run.py

```diff
@@ -279,6 +286,52 @@ def generate_popsycle_config_file(radius_cut, obs_time,
     generate_config_file(config_filename, config)
 
 
+def generate_pbh_config_file(config_filename, fdm, pbh_mass, r_max, c, r_vir, inner_slope, v_esc):
```

Comment:
Given the new method for inputing mass model, I think we should rewrite this function.

Reply:
Also done, in next push